lib/rouge/demos/turing

```diff
@@ -0,0 +1,38 @@
+var myStr : string := "hello"
```

Comment:
Please trim this down a bit. It's meant to be a small visual demo of what the lexer does, not of the language per se, for use e.g. on http://rouge.jneen.net/ - Most are 10-25 lines or so.